src/api/models.py

```diff
@@ -136,6 +142,14 @@ class User(Base):
                                  secondary='user_activity',
                                  lazy='dynamic',
                                  backref='participants')
+    roles = db.relationship('Role', secondary='user_role', backref='user')
```

Comment:
Any context on why these roles, relationship is on the Base model?

Reply:
It's actually on the User model @georgreen

Reply:
@Andretalik This can't be put here, because all the models inherit this base model behaviour, meaning that all models have a role relationship if this implementation is upheld.

Reply:
@georgreen Github has truncated out some of the code. That relationship is under the User class.